### PR TITLE
Supersched

### PR DESCRIPTION
**Pull request by mcoughlin**
**Does this pull request make any changes to the database?**
just db create to add the default args.

Code changes that affect the database require special attention for data migration.
If your change affects the database, describe the migration plan here.

### CODE REVIEW

File: growth/too/templates/plan_new.html [hunk 1]
```diff
@@ -71,7 +71,13 @@
                         <small id="balanceHelpBlock" class="form-text text-muted">
                         Require field observations in all requested filters
                         </small>
-
+                    </div>
+                    <div class="custom-control custom-switch" aria-describedby="balanceHelpBlock">
+                        {{ render_field(form.planned, class='custom-control-input') }}
+                        <label for=planned class=custom-control-label>Use tonight's plan</label>
```
Comment (shreyasahasram08) [inline growth/too/templates/plan_new.html]: minor suggestion: 'Account for tonight's plan'
Also, would it be possible for you to add a short description to the use previous flag (mainly to help differentiate from doPlanned flag)? Observation planning is starting to get a bit complicated now on the user side :)

File: growth/too/templates/plan_new.html [hunk 2]
```diff
@@ -149,6 +155,23 @@
...
+                    <label for=completed_window_start>Start time (completed observations)</label>
+                    {{ render_field(form.completed_window_start) }}
+                </div>
+                <div class="form-group col-sm-6">
+                    <label for=completed_window_end>End time (completed observations)</label>
+                    {{ render_field(form.completed_window_end) }}
+                </div>
+                <div class="custom-control custom-switch" aria-describedby="balanceHelpBlock">
+                    {{ render_field(form.completed, class='custom-control-input') }}
+                    <label for=completed class=custom-control-label>Accounted for completed observations</label>
```
Comment (shreyasahasram08) [inline growth/too/templates/plan_new.html]: minor suggestion: 'Account for completed observations'

**Comment by lpsinger** [comment]
Aaaaaa! I have no idea what these changes do and they _definitely_ require a database rebuild.

**Comment by shreyasahasram08** [comment]
Do you want me to revert this?

**Comment by mcoughlin** [comment]
it should just be a db create here.

**Pull request merged**